pkg/admin/data/data_manager.go

```diff
@@ -181,6 +195,28 @@ func (dm *manager) RemoveData() error {
 	return nil
 }
 
+// isMicroShiftBackup verifies if given path is a valid MicroShift backup
+// by checking if all expected subdirs exists
+func (dm *manager) isMicroShiftBackup(path string) error {
```

Comment:
This function could use some unit tests.

Reply:
Had to split the function because it interacts with the filesystem, but the functional part is now unit-tested